nostrudel/umbrel-app.yml

```diff
@@ -0,0 +1,42 @@
+manifestVersion: 1
+id: nostrudel
+category: social
+name: noStrudel
+version: "0.37.1"
+tagline: A sandbox for exploring nostr
+icon: https://raw.githubusercontent.com/hzrd149/umbrel-community-app-store/master/hzrd149-nostrudel/icon.svg
```

Comment:
This line can be removed completely. We will host the icon here:
https://github.com/getumbrel/umbrel-apps-gallery

Reply:
removed :+1: